src/modules/upcoming-events.module/module.html

```diff
@@ -3,5 +3,6 @@
 
 <div class="upcoming-events">
     <h2>Upcoming Events</h2>
-    <div id="upcoming-events__module" data-portal-id="{{ portal_id }}"></div>
+    {% set appSlug = content_by_id(module.event_page).slug %}
```

Comment:
I think this is going to get weird for index pages, `/`, which would result in links to `//my-event-2`.

What happens when the selected `event_page` is on a different domain than the upcoming-events module?

Reply:
hmmm.. Those are both really good questions that I think we'll need to think through.

Reply:
I think using `absolute_url` is the way to go.